relayer-cli/src/lib.rs

```diff
@@ -18,6 +18,7 @@
 
 pub mod application;
 pub mod commands;
+pub(crate) mod conclude;
```

Comment:
I actually wanted to ask this before, why don't we use the crate scope more often. I believe in this case for example all mods, except application, can be public at crate level.

Reply:
Agree, for 0.1.0 this may be relevant. @romac any thoughts on this (I can add an issue if so) or is it low-priority?

Reply:
I don't thin it matters too much for the `relayer-cli` crate, but yeah in general it wouldn't hurt either to make more modules crate scoped, to limit the surface area of our libs.